modules/programs/bat.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.bat;
+
+in
+
+{
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.bat = {
+    enable = mkEnableOption "Bat";
+
+    package = mkOption {
```

Comment:
As far as I can tell there is only one bat package in Nixpkgs so perhaps this option is unnecessary?